its/tests/test_analysis.py

```diff
@@ -27,3 +27,30 @@ def test_minimal_project(sonarqube_client: SonarQubeClient):
     response = sonarqube_client.get_project_issues("minimal")
     data = response.json()
     assert len(data["issues"]) == 1
+    analyses_data = sonarqube_client.get_project_analyses("minimal").json()
+    latest_analysis_data = analyses_data['analyses'][0]
+    assert latest_analysis_data['projectVersion'] == '1.2'
+
+
+def test_minimal_project_read_poetry_data(sonarqube_client: SonarQubeClient):
+    process = CliClient().run_analysis(sonarqube_client, params=["-read.project.config"], sources_dir="minimal")
+    assert process.returncode == 0
+    response = sonarqube_client.get_project_issues("minimal")
+    data = response.json()
+    assert len(data["issues"]) == 1
+    analyses_data = sonarqube_client.get_project_analyses("minimal").json()
+    latest_analysis_data = analyses_data['analyses'][0]
+    assert latest_analysis_data['projectVersion'] == '1.5'
+
+
+def test_minimal_project_unexpected_arg(sonarqube_client: SonarQubeClient):
+    process = CliClient().run_analysis(sonarqube_client, params=["-unexpected"], sources_dir="minimal")
+    assert process.returncode == 0
+    assert "ERROR: Unrecognized option: -unexpected" in process.stdout
```

Comment:
I was wondering if the sonar cli was writing to the STDERR stream when an error is encountered. If this is the case we could check that STDERR is empty or assert on errors. But I think for now this is totally fine like this! It is just an idea for the next iteration linked to [this ticket](https://sonarsource.atlassian.net/browse/PYSCAN-53)